clients/fides-js/src/components/fides.css

```diff
@@ -253,17 +257,6 @@ div#fides-button-group {
   z-index: 5;
 }
 
-div.fides-acknowledge-button-container {
-  margin-inline: var(--fides-overlay-padding);
-  margin-bottom: var(--fides-overlay-padding);
-  display: flex;
-  justify-content: end;
-}
-
-div.fides-banner-acknowledge .fides-banner-button {
-  max-width: 168px;
-}
-
```

Comment:
CSS changes always make me uneasy as we can't confirm 100% the changes  we make don't affect our customers. Do we have to remove existing CSS to get the new acknowledge button working correctly? Just curious if we can minimize changes at all to the classes that already exist

Reply:
From what I could tell, `fides-acknowledge-button-container` didn't exist and `.fides-banner-button` was never a child of `.fides-banner-acknowledge` so neither of these styles were ever applied. This wasn't so much of a change as it was a fix with the new styles added below along with piggy-backing on the "first button" logic/styles for the acknowledge button.

Reply:
ignore my last comment. I just double checked this stuff...

The reason for this change is that the old/existing Acknowledge button did not include the language switcher, or the manage and privacy links. It was just an "Ok" button by itself.

![CleanShot 2024-07-30 at 15 54 25@2x](https://github.com/user-attachments/assets/95a8218d-bae0-408a-8808-37661ac19a64)

When I noticed that, I checked with Jack who confirmed that the only change should be the Opt In/Out buttons should be replaced with the Acknowledge button.

These styles were supporting the standalone button only.

To fix it, I moved the Acknowledge button as part of the "first button" (ie. Save Button) logic instead, which is integrated with the actions to the left already. The "first button" styles apply now, and these are obsolete.

Any custom styles would have already been applied to "first button" so I feel confident in this update, especially since it now correctly includes the fix.

Reply:
Ohh ok this makes sense, thanks for explaining!